.github/workflows/build.yaml

```diff
@@ -79,7 +79,7 @@ jobs:
       run: cosign sign --key env://COSIGN_PRIVATE_KEY ${{ env.image_digest_sha }} ${{ env.image_digest_branch }} -y
       env:
         COSIGN_PASSWORD: ${{ secrets.COSIGN_PRIVATE_KEY_PASSWORD }}
-        COSIGN_PRIVATE_KEY: ${{ secrets.COSIGN_PRIVATE_KEY }}      
```

Comment:
While this change is nice, this is creep. It is not relevant to updating the ZDV URL.